models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatRequestTests.java

```diff
@@ -29,17 +29,18 @@
  * @author Christian Tzolov
  * @author Thomas Vitale
  */
-public class OllamaChatRequestTests {
+class OllamaChatRequestTests {
 
 	OllamaChatModel chatModel = OllamaChatModel.builder()
 		.ollamaApi(new OllamaApi())
 		.defaultOptions(OllamaOptions.builder().model("MODEL_NAME").topK(99).temperature(66.6).numGPU(1).build())
 		.build();
 
 	@Test
-	public void createRequestWithDefaultOptions() {
+	void createRequestWithDefaultOptions() {
```

Comment:
These are the only Ollama tests that I had to change, and just because they are testing implementation details and not the public API. All the other tests I left untouched, supporting the backward compatibility validation.